### DISCUSSION TITLE
NFT音樂作品

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#287] [General]
https://www.bnext.com.tw/article/62954/nft-music

台灣歌手周興哲的作品以2.8萬美元售出，令人期待NFT這股風潮未來的走向。

**Comment by chun1203** [comment]
NFT對於創作者來說真的是一大福音

**Comment by JAIMEHUNG** [comment]
能讓創作者得到收益，以應對傳統市場的萎縮

**Comment by 107602505** [comment]
現在大家聽音樂，打開串流平台選擇非常多，過去一首歌的利潤要分散給音樂廠牌、版權機構、代理發行商等，且每首歌被點擊，創作者平均只能獲得約0.03美元的分潤，壓縮獲利空間，而NFT的出現保障了創作者的獲利，也能強化跟粉絲之間的連結。

**Comment by zixunn** [comment]
NFT的出現成功讓創作的價值再成倍增長了。

**Comment by cody910120** [comment]
真希望我可以用NFT賺錢，但我們現在看到的都是成功的案例而沒注意到那大多數失敗的案例，投資有風險阿你各位

**Comment by YuHengChen54** [comment]
現在各大創作者都開始把作品用成NFT，多賺了好多錢

**Comment by 109302013evantsao** [comment]
NFT對我來說就像買一個我只看的到但摸不到的東西，我個人還是比較喜歡實體收藏，應該還是不會去踏入NFT

**Comment by BryanNee** [comment]
不知道NFT會不會一夕之間崩盤
